src/generic_ui/polymer/copypaste.html

```diff
@@ -48,10 +23,8 @@
       color: white;
     }
     </style>
-    <div id='cptoolbar'>
-      <div class='prevArrow {{ { disabled: ui.copyPasteGettingState === GettingState.GETTING_ACCESS || ui.copyPasteSharingState === SharingState.SHARING_ACCESS } | tokenList }}'>
-        <core-icon icon="arrow-back" on-tap='{{prev}}'></core-icon>
-      </div>
+
+    <uproxy-app-bar disableback='{{ ui.copyPasteGettingState === GettingState.GETTING_ACCESS || ui.copyPasteSharingState === SharingState.SHARING_ACCESS }}' on-go-back='{{ prev }}'>
```

Comment:
Does this conflict with the other new CL where instead of disabling the back button on copy+paste, we instead let the user click back but show them a warning that it will cancel their copy+paste proxy?